keyvault/src/main/java/com/microsoft/identity/internal/test/keyvault/ApiClient.java

```diff
@@ -93,6 +93,9 @@ public ApiClient() {
         authentications.put("oauth", oAuthAuthentication);
         // Prevent the authentications from being modified.
         authentications = Collections.unmodifiableMap(authentications);
+        List<Protocol> protocols = new ArrayList<>();
```

Comment:
I'd suggest adding a comment here around why this is necessary... mostly so no one removes without testing.

Reply:
Addressed